Lib/test/list_tests.py

```diff
@@ -363,6 +365,8 @@ def __eq__(self, other):
             # verify that original order and values are retained.
             self.assertIs(x, y)
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
     def test_index(self):
         super().test_index()
```

Comment:
In the future, classes dependent on this test may start unexpectedly passing. Will they unexpectedly pass _all at once_ or _one at a time_?

Reply:
Tests are supportive code. They will be handled when they actually be solved.

The answer is I don't know. Sometimes they passes at once, sometimes not. I didn't look in the details.